### PR TITLE
Eth 98 binance adapter

### PR DESCRIPTION
**Pull request by Isentrope1**

### CODE REVIEW

File: packages/client/test/integration/dataunion/binanceAdapter.test.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { BigNumber, Contract, providers, Wallet } from 'ethers'
```
Comment (jtakalai) [inline packages/client/test/integration/dataunion/binanceAdapter.test.ts:1]: Lots of linter errors, please fix lint issues (run `npm run eslint`)

Comment (jtakalai) [reply packages/client/test/integration/dataunion/binanceAdapter.test.ts:1]: at least IDEA and VScode have "fix all eslint problems" functionality that mostly fixes them immediately.

**Review by jtakalai — CHANGES_REQUESTED** [review]

File: packages/client/test/integration/dataunion/binanceAdapter.test.ts (hunk 1 above)
Comment (isentropy) [reply packages/client/test/integration/dataunion/binanceAdapter.test.ts:1]: ok Ill try

Comment (jtakalai) [reply packages/client/test/integration/dataunion/binanceAdapter.test.ts]: 👍

**Review by jtakalai — APPROVED** [review]

**Comment by jtakalai** [comment]
failing tests are not related to data unions

**Pull request merged**